### PR TITLE
Add gnocchi.conf

### PR DESCRIPTION
**Pull request by chenlizhong**

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -1811,6 +1811,16 @@
...
         },
+        {
+            "file": "/var/lib/config-data/gnocchi/etc/gnocchi/gnocchi.conf",
+            "pattern": [],
+            "symbolic_name": "gnocchi_conf"
+        },
+        {
+            "file": "/etc/gnocchi/gnocchi.conf",
+            "pattern": [],
+            "symbolic_name": "gnocchi_conf"
```
Comment (bfahr) [inline uploader.json]: @chenlizhong you will need to add the filter from your plugin to the specs in this PR.

**Comment by eduardocerqueira** [comment]
There was an infra issue Yesterday that was the cause for the CI tests failed. more info about this at #insights-qe channel. Test will run again.

**Comment by eduardocerqueira** [comment]
test this please

**Comment by chenlizhong** [comment]
test this please

**Pull request merged**